src/Compilers/Core/Portable/Operations/IFlowAnonymousFunctionOperation.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
```

Comment:
Should move to names to FlowAnalysis namespace - I can do it once this is merged.

Reply:
> Should move to FlowAnalysis namespace - I can do it once this is merged.

I don't think the namespace is in the branch yet.

---
In reply to: [192234234](https://github.com/dotnet/roslyn/pull/27282#discussion_r192234234) [](ancestors = 192234234)